master/getting-started/kubernetes/installation/hardway/install-typha.md

```diff
@@ -0,0 +1,317 @@
+          periodSeconds: 10
+        volumeMounts:
+        - name: calico-typha-ca
+          mountPath: "/calico-typha-ca"
+          readOnly: true
+        - name: calico-typha-certs
+          mountPath: "/calico-typha-certs"
+          readOnly: true
+      volumes:
+      - name: calico-typha-ca
+        configMap:
+          name: calico-typha-ca
+      - name: calico-typha-certs
+        secret:
+          secretName: calico-typha-certs
+EOF
+```
+
+We set `TYPHA_CLIENTCN` to `calico-felix` which is the common name we will use on the certificate Felix will use in the
+next lab.
+
+Verify Typha is up an running with 3 instances
+
+```
+kubectl get pods -l k8s-app=calico-typha -n kube-system
+```
+
+Result:
+
+```
+NAME                            READY   STATUS    RESTARTS   AGE
+calico-typha-66498ddfbd-2pzsr   1/1     Running   0          69s
+calico-typha-66498ddfbd-lrtzw   1/1     Running   0          50s
+calico-typha-66498ddfbd-scckd   1/1     Running   0          62s
+```
+{: .no-select-button}
+
+## Install Service
+
+Felix uses a Kubernetes Service to get load-balanced access to Typha.
+
+```
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: calico-typha
+  namespace: kube-system
+  labels:
+    k8s-app: calico-typha
+spec:
+  ports:
+    - port: 5473
+      protocol: TCP
+      targetPort: calico-typha
+      name: calico-typha
+  selector:
+    k8s-app: calico-typha
+EOF
+```
+
+Validate that Typha is using TLS.
+
+```
+TYPHA_CLUSTERIP=$(kubectl get svc -n kube-system calico-typha -o jsonpath='{.spec.clusterIP}')
+curl https://$TYPHA_CLUSTERIP:5473 -v --cacert typhaca.crt
+```
+
+Result
+```
+* Rebuilt URL to: https://10.103.120.116:5473/
+*   Trying 10.103.120.116...
+* TCP_NODELAY set
+* Connected to 10.103.120.116 (10.103.120.116) port 5473 (#0)
+* ALPN, offering h2
+* ALPN, offering http/1.1
+* successfully set certificate verify locations:
+*   CAfile: typhaca.crt
+  CApath: /etc/ssl/certs
+* (304) (OUT), TLS handshake, Client hello (1):
+* (304) (IN), TLS handshake, Server hello (2):
+* TLSv1.2 (IN), TLS handshake, Certificate (11):
+* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
+* TLSv1.2 (IN), TLS handshake, Request CERT (13):
+* TLSv1.2 (IN), TLS handshake, Server finished (14):
+* TLSv1.2 (OUT), TLS handshake, Certificate (11):
+* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
+* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
+* TLSv1.2 (OUT), TLS handshake, Finished (20):
+* TLSv1.2 (IN), TLS alert, Server hello (2):
+* error:14094412:SSL routines:ssl3_read_bytes:sslv3 alert bad certificate
+* stopped the pause stream!
+* Closing connection 0
+curl: (35) error:14094412:SSL routines:ssl3_read_bytes:sslv3 alert bad certificate
+```
+{: .no-select-button}
+
+This demonstrates that Typha is presenting its TLS certificate and rejecting our connection because we do not present a certificate.
+In the next lab we will deploy Felix with a certificate Typha will accept.
```

Comment:
We call it "Felix" throughout this doc, but then continue to discuss installing "Node" (which also includes confd, which we do mention briefly at the top).

What do you think about just calling it "calico/node" throughout to be consistent and a bit more accurate?